csv/ParameterDictionary.csv

```diff
@@ -26,7 +26,7 @@ TRHPH_A,DICT47,,trhph_sample,"PD7,PD10,PD11,PD12,PD16,PD421,PD422,PD423,PD424,PD
 VEL3D_C,DICT56,,vel3d_cd_hardware_configuration,"PD7,PD10,PD11,PD12,PD16,PD863,PD461,PD462,PD463,PD464,PD465,PD466,PD467,PD468,PD113",PD7,,Engineering,Hardware Configuration,
 VEL3D_C,DICT57,,vel3d_cd_head_configuration,"PD7,PD10,PD11,PD12,PD16,PD863,PD470,PD471,PD472,PD473,PD474,PD475,PD476,PD477,PD478",PD7,,Engineering,Header Configuration,
 VEL3D_C,DICT58,,vel3d_cd_user_configuration,"PD7,PD10,PD11,PD12,PD16,PD863,PD479,PD480,PD481,PD482,PD483,PD484,PD485,PD478,PD487,PD488,PD489,PD490,PD491,PD492,PD493,PD494,PD495,PD496,PD497,PD498,PD499,PD500,PD501,PD502,PD503,PD504,PD505,PD506,PD507,PD508,PD509,PD510,PD511,PD512,PD513,PD514,PD515,PD516,PD517,PD518,PD519,PD520,PD521,PD522,PD523,PD524,PD526,PD525,PD527,PD528,PD529,PD530,PD531,PD532,PD533,PD1088,PD534",PD7,,Engineering,User Configuration,
-"OPTAA,OPTAA_CD",DICT61,OPTAA,optaa_sample,"PD7,PD10,PD11,PD12,PD16,PD19,PD20,PD863,PD583,PD584,PD585,PD312,PD586,PD587,PD588,PD589,PD590,PD591,PD592,PD593,PD594,PD595,PD596,PD597,PD598,PD743",PD7,,Science,Data Products,ready for review
+"OPTAA,OPTAA_CD",DICT61,OPTAA,optaa_sample,"PD7,PD10,PD11,PD12,PD16,PD19,PD20,PD863,PD583,PD584,PD585,PD312,PD586,PD587,PD588,PD589,PD590,PD591,PD592,PD593,PD594,PD595,PD596,PD597,PD598,PD743,PD8080,PD8081,PD8082",PD7,,Science,Data Products,ready for review
```

Comment:
It's important to note that these are derived parameters, otherwise a change to an existing stream would be a breaking change.

Reply:
I've noted that the new parameters as derived in the release notes file.  Should I note this elsewhere?

Reply:
No need. I just want people in these review to keep this in mind (so I'm not the only one thinking about it).